docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,104 @@
+# Bot comparison
+
```

Comment:
I am not sure if it is worth mentioning it, but Dependabot is written in Ruby while Renovate is in Javascript 

For users looking to be future contributors or to extend the bots that might matter